src/shared/constants/tell-me-questions/tell-me-questions.cat-adi-part2.constants.ts

```diff
@@ -0,0 +1,37 @@
+/* tslint:disable */
+import { VehicleChecksQuestion } from '../../../providers/question/vehicle-checks-question.model';
+
+export default [
+  {
+    code: 'T01',
```

Comment:
We shouldn't have any 0 in the codes , should be T1, T2

Reply:
This is how B+E does it... is that a mistake?

Reply:
Yes, it was a mistake. We can't change them on B+E due to it being live and the issues it may cause